### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by celoy** [C0PEP0D/sheld0n#3]
I followed the tutorial and had a problem when creating a new equation for buoyant particles. 

$ ./create_new_equation buoyant_particle

Traceback (most recent call last):
  File "/Users/Ch/Documents/Python/sheld0n/cases/case00/param/solutions/./create_new_equation", line 5, in <module>
    import argcomplete
ModuleNotFoundError: No module named 'argcomplete'

**Comment by rmonthil** [comment]
Indeed, you should just need to install the python module `argcomplete` it should be fine. I'll add it to the dependencies.
